packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -31,9 +31,14 @@ import {
  * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
  * Please do not modify existing entries for backwards compatibility.
  */
- export enum TelemetryDataTag {
-    /** Data containing terms from code packages that may have been dynamically loaded */
+export enum TelemetryDataTag {
+    /**
+     * Data containing terms from code packages that may have been dynamically loaded
+     * @deprecated 1.0, will be removed in next release (see issue #6603). Use `TelemetryDataTag.CodeArtifact` instead.
```

Comment:
Not sure why it lists 1.0 as a version, that's client only change, so should use current client version?

Reply:
It was mentioned that 0.59 was already cut and `next` is pushing to 1.0. Should I be noting 0.59 in this comment and then noting this change under 1.0 in `BREAKING.md`?

Reply:
Oh, I did not realize next one is 1.0. Probably, yes? @anthony-murphy is poking this question, so maybe we should ask him?
I think it's fine either way, we just need to double check the actual version number when next cut is made and potentially change it if plans change

Reply:
@anthony-murphy Waiting for confirmation if `1.0` is the correct version to list here